packages/x-components/src/plugins/__tests__/x-plugin-emitters.spec.ts

```diff
@@ -211,7 +211,7 @@ describe('testing X Plugin emitters', () => {
 
       /* Emitters relies on Vue watcher that are async. We need to wait a cycle before testing if
        they have emitted or not. */
-      await Promise.resolve();
+      jest.advanceTimersByTime(0);
```

Comment:
As we are testing a `XXXChanged` event, it is only debounced once. So running all the timers once is enough for the test